docs/notebooks/multi_objective_ehvi.pct.py

```diff
@@ -214,6 +214,69 @@ def log_hv(observations):
 ax.set_ylabel("log HV difference")
 plt.show()
 
+
+# %% [markdown]
+# ## Batch multi-objective optimization
+#
+# EHVI can be extended to the case of batches (i.e. query several points at a time) using the `FantasizeAcquisitionFunction`. `FantasizeAcquisitionFunction` works by greedily optimising a base acquisition function, then "fantasizing" the observations at the chosen query points and updating the predictive equations of the models as if the fantasized data was added to the models. The only changes that need to be done here are to wrap the `ExpectedHypervolumeImprovement` in a `FantasizeAcquisitionFunction` object, and set the rule argument `num_query_points` to a value greater than one. Here, we choose 10 batches of size 3, so the observation budget is the same as before.
+
+# %%
+model = build_stacked_independent_objectives_model(initial_data, num_objective)
+
+from trieste.acquisition.function import FantasizeAcquisitionFunction
+
+batch_ehvi = FantasizeAcquisitionFunction(ExpectedHypervolumeImprovement())
+batch_rule: EfficientGlobalOptimization = EfficientGlobalOptimization(
+    builder=batch_ehvi, num_query_points=3
+)
+num_steps = 2
```

Comment:
should be 10 as written in the text, no?

Reply:
also still wip, will update later

Reply:
back to normal values